logic/common/tiled_utils.py

```diff
@@ -159,10 +159,29 @@ def RotateTileId(tile_id):
     0b001: 0b100
 }
 
+
+
+#--------------------------------------------------#
+'''Object Properties'''
+
+def GetPolyPoints( line_vertice ):
+    '''Convert string into list of points: [ [x1,y1], [x2,y2], ... ]'''
+    list_points = line_vertice.split(" ")
+    list_condensed = [ pt.split(",") for pt in list_points]
+    list_num = [ [float(v) for v in pt] for pt in list_condensed ]
+    return list_num
```

Comment:
Let's use tuples. See my trello comment

Next, I'd like to refactor the names used throughout this function.

Take the argument name "**line_vertice**" for instance. It would throw people off because a "vertice" (singular) is one point (so 1 pair of numbers). It doesn't capture that this is actually multiple points and that they're strings.

Or take the return value "**list_num**". list_num would make me think it'd have the form [1, 2, 3, 4]. It doesn't capture that it's a 'list of tuple/lists of nums".

Let's rename this function to "ConvertTiledPolyPoints" and then give it a more expanded definition similar to TrimTiles2d() in this file (ie, let's give a usage example).

```
def ConvertTiledPolyPoints(input_string):
    point_strings = input_string.split()
    point_pair_strings = [point_string.split(',') for point_string in point_strings]
    poly_points = [(float(x), float(y)) for x, y in point_pair_strings]
    return poly_points
```